src/java.base/share/classes/java/text/DateFormat.java

```diff
@@ -746,6 +746,10 @@ public TimeZone getTimeZone()
      * <p>This leniency value is overwritten by a call to {@link
      * #setCalendar(java.util.Calendar) setCalendar()}.
      *
+     * @implSpec A {@link Character#SPACE_SEPARATOR SPACE_SEPARATOR} in the input
+     * text will match any other {@link Character#SPACE_SEPARATOR SPACE_SEPARATOR}s
+     * in the pattern with the lenient parsing; otherwise, it will not match.
```

Comment:
LGTM.

Nit: Might read better as either "in the pattern with lenient parsing" or "in the pattern when parsing is lenient"

Reply:
Thanks. Fixed.